dotnet/README.md

```diff
@@ -9,4 +9,8 @@ You'll note that instrumentation is done differently depending on the project:
 * `year-service` is a service built as a minimal .NET 6 API that uses the standard OpenTelemetry libraries to configure instrumentation. It's more lines of code.
 * `name-service` is a service built with .NET 5 that uses the standard OpenTelemetry libraries.
 
-The goal here is to demonstrate that you can configure instrumentation in several ways with .NET but still send data to Honeycomb.
\ No newline at end of file
+The goal here is to demonstrate that you can configure instrumentation in several ways with .NET but still send data to Honeycomb.
+
+### Run the services

```

Comment:
There's a dotnet section in `DEVELOPING.md` that can probably change to match node, where other details are removed and it just says to view the language-specific setup instructions in the readme.

Also, maybe we should add to `tilt down dotnet` twice to kill the services?